.gitattributes

```diff
@@ -0,0 +1 @@
+/composer.lock -diff
```

Comment:
👎

It can be useful to be able to view lock file changes, as they're the record of what's really installed.

Reply:
You can still view them with `git diff --text`. This just turns them off by default because they're very long and 99% of the time not what you want to look at.

Reply:
What about `git add -p`? What happens there? I don't think we should assume developers are Git pros. Is there something you can set in your own config to hide things like this?